docs/developing-locally.rst

```diff
@@ -9,24 +9,53 @@ Setting Up Development Environment
 
 Make sure to have the following on your host:
 
-* virtualenv_;
-* pip;
-* PostgreSQL.
+* Python 3.6
+* PostgreSQL_.
+* Redis_, if using Celery
 
 First things first.
 
-#. `Create a virtualenv`_.
+#. Create a virtualenv: ::
 
-#. Activate the virtualenv you have just created.
+    $ python3 -m venv <virtual env path>
```

Comment:
I just realised that `python3` might point to Python 3.7, which isn't supported yet (due to Celery, at least).

Reply:
why not use `virtualenv <path/to/new/virtualenv/> --python=python3.6`

Reply:
This one is included in [python since version 3.3](https://docs.python.org/3.6/library/venv.html), so that's one fewer thing to install, no need to have a requirement on the standalone `virtualenv` library.

Correct me if I'm wrong, but I think they do the same thing.

Reply:
I think you are right. We can get rid of virtualenv. They basically do the same thing but is `venv` is a built-in module.

https://www.reddit.com/r/learnpython/comments/4hsudz/pyvenv_vs_virtualenv/

So we can use `python3.6 -m <virtual env path>` or `/path/to/python3.6 -m <virtual env path>`